src/test/groovy/org/gradle/android/SimpleAndroidApp.groovy

```diff
@@ -115,6 +115,11 @@ class SimpleAndroidApp {
             android {
                 compileSdkVersion 26
                 buildToolsVersion "26.0.2"
+                dataBinding.enabled = true
```

Comment:
So the idea is that, even with databinding, most of the tasks are still loaded from cache, right?
Should we make that two tests, one with databinding enabled and one without?

Reply:
Good idea. Let's do that when we come back to this PR (if at all necessary).